software/debugging/index.rst

```diff
@@ -0,0 +1,455 @@
+
+    .. code-block:: bash
+
+           export ALLINEA_CONFIG_DIR=<Somewhere on the Filesystem that can be accessed by the compute nodes i.e. /lustre/orion/<project>>
+           export ALLINEA_REVERSE_CONNECT_DIR=<Somewhere on the Filesystem that can be accessed by the compute nodes i.e. /lustre/orion/<project>>
+
+
+    Also, if you plan on running the Forge client from your local machine (i.e. laptop), you will need to create a bash file containing the above environment vars. The file can be saved in ``/ccs/home/<user>``. Once created and saved, you will
+    enter the path to the file in the Forge Remote Launch setup window next to ``Remote Script`` as shown below.
+
+    .. figure:: /images/ddt_remote_script.png
+           :align: center
+           :width: 800
+
+    #. Launch the Forge remote client and connect to a remote host using the steps above. Once connected, this client will monitor for new connections.
+
+    .. figure:: /images/ddt_launch_frontier.png
+        :align: center
+        :width: 400
+
+    #. In a separate terminal, load the ``forge/22.1.1`` module, and run a ``ddt --connect`` command via the batch system (e.g. by editing and running a job script, or running with an interactive shell).
+
+        .. code-block:: bash
+
+            module load forge/22.1.1
+            ddt --connect srun -n 8 ./hello_mpi_omp
+
+    #. The remote client will notify you of a new connection.
+
+    .. figure:: /images/ddt_reverse_prompt_frontier.png
+        :align: center
+        :width: 400
+
+    #. Once accepted, you can configure some final debugging options before launching the program.
+
+    .. figure:: /images/ddt_run_options_frontier.png
+        :align: center
+        :width: 600
+
+    #. Click “Run”, and DDT will start your session.
+
+*******
+GNU GDB
+*******
+
+`GDB <https://www.gnu.org/software/gdb/>`__, the GNU Project Debugger,
+is a command-line debugger useful for traditional debugging and
+investigating code crashes. GDB lets you debug programs written in Ada,
+C, C++, Objective-C, Pascal (and many other languages).
+
+More information on its use on OLCF systems can be found below.
+
+.. tabbed:: Summit
+
+    GDB is available on Summit under all compiler families:
+
+    .. code::
+
+        module load gdb
+
+    To use GDB to debug your application run:
+
+    .. code::
+
+        gdb ./path_to_executable
+
+    Additional information about GDB usage can befound on the `GDB Documentation Page <https://www.sourceware.org/gdb/documentation/>`__.
+
+.. tabbed:: Andes
+
+    GDB is availableon andes via the ``gdb`` module:
+
+    .. code::
+
+        module load gdb
+
+    To use GDB to debug your application run:
+
+    .. code::
+
+        gdb ./path_to_executable
+
+    Additional information about GDB usage can befound on the `GDB Documentation Page <https://www.sourceware.org/gdb/documentation/>`__.
+
+.. tabbed:: Frontier
+
+    GDB is availableon Summit under all compiler families:
```

Comment:
Typo on 357: "availableon" --> "available on"

Reply:
Oh and you said Summit instead of Frontier for this one, too

Reply:
fixed in d7aa0d08fd41549a2e8a06f3c9d854ca78040d58